webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/StatusReports.scala

```diff
@@ -106,12 +106,12 @@ object RunComplianceInfo {
   final case class PolicyModeInconsistency(problems: List[PolicyModeError]) extends RunComplianceInfo
 }
 
-final class NodeStatusReport private (
-    val nodeId:     NodeId,
-    val runInfo:    RunAndConfigInfo,
-    val statusInfo: RunComplianceInfo,
-    val overrides:  List[OverridenPolicy],
-    val reports:    Map[PolicyTypeName, AggregatedStatusReport]
+final case class NodeStatusReport(
```

Comment:
I see not longer benefits to make that constructor private: the `reports` type is now different for when we need to compute it with the apply method and what is in the class. 

Plus, it will ease the usage.